content/develop/connect/insight/rdi-connector.md

```diff
@@ -0,0 +1,158 @@
+
+- YAML files in the configuration and job file editors
+- JMESPath and SQL function snippets in a dedicated editor. To open the JMESPath and SQL editor, click the **SQL and JMESPathEditor** button as shown above. A new editor window will open in the lower half of the screen.
+
+For example, if you decided to write your own configuration pipeline without using a template, you would see auto-completion prompts such as the one shown below.
+
+<img src="../images/ri-rdi-pl-editor-autoc.png">
+
+While this isn't a replacement for the RDI documentation, it can help speed you along after you have basic familiarity with the building blocks of RDI pipeline files.
+
+Here's an example showing the SQL and JMESPath editor pane. Note the toggle in the bottom left corner of this editor pane. Clicking it allows you to select from:
+
+- SQLite functions
+- JMESPath
+
+After constructing your SQLite or JMESPath code, copy it to the main editor window.
+
+<img src="../images/ri-rdi-pl-editor-sql-minie.png">
+
+## Dry run transformation job pipelines
+
+After you've created a transformation job pipeline, you can execute a dry run on the RDI server. To do that, click on **Dry Run** in the lower right side of the editor pane. A new **Test transformation logic** panel will open to the side. There are two vertically-stacked panes: **Input** and **Results**. In the **Input** section, enter JSON data that will trigger the transformation. Any results will be displayed in the **Results** section.
+
+There are two tabs in the **Results** section:
+
+1. **Transformations** - this is where you'll see JSON output from your dry run.
+1. **Output** - (not shown) this is where you'll see the Redis commands that would have been run in a real scenario.
+
+Here's an example.
+
+<img src="../images/ri-rdi-pl-dryrun.png">
+
+## Deploy pipelines and view statistics
+
+If you're satisfied with your configuration and transformation job pipelines, you can deploy them to the RDI management plane. Click the **Deploy Pipeline** button to proceed.
+
+After your pipelines have been deployed, you can view various statistics for your RDI deployment. To do so, click  the **Pipeline Status** menu button in the left side menu panel.
+
+<img src="../images/ri-rdi-stats-view.png">
+
+Each statistics section is either static or refreshed automatically at a particular interval that you set.
+The first section, **Processing performance information** is set by default to refresh every 5 seconds.
+The other sections are static and need to be refreshed manually by pressing the refresh button at the top right of each section.
+You can also set up automatic refresh for the other sections.
+
+To set up automatic refresh for one or more statistics sections, click on the downward arrow at the end of the **Last refresh** line.
+Then enable the **Auto Refresh** setting and set your desired refresh interval in seconds. This is shown in the previous image.
```

Comment:
the next step can be opening a Redis database in Redis Insight to check the real keys created.

Reply:
Done. Look for next merge.